azext_iot/central/services/device.py

```diff
@@ -616,3 +616,36 @@ def run_manual_failback(
     _utility.log_response_debug(response=response, logger=logger)
 
     return _utility.try_extract_result(response)
+
+
+def purge_c2d_messages(
+    cmd,
+    app_id: str,
+    device_id: str,
+    token: str,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+) :
+    """
+    Purges cloud to device (C2D) message queue for the specified device.
+
+    Args:
+        cmd: command passed into az
+        app_id: name of app (used for forming request URL)
+        device_id: unique case-sensitive device id,
+        token: (OPTIONAL) authorization token to fetch device details from IoTC.
+            MUST INCLUDE type (e.g. 'SharedAccessToken ...', 'Bearer ...')
+        central_dns_suffix: {centralDnsSuffixInPath} as found in docs
+
+    Returns:
+        {
+            message: 'Cloud to device (C2D) message queue purged for device {device_id}.\\n
+            Total messages purged: {totalMessagesPurged}.'
+        } on success
+        Raises error on failure
+    """
+    url = "https://{}.{}/{}/{}/c2d".format(
```

Comment:
API version is not being used here, does the corresponding param need to be exposed to users?

Reply:
no, there is no versioning for following api's

Reply:
Ok. So presumably we can remove the API version at the command level to simplify the command surface.

![image](https://user-images.githubusercontent.com/28658112/148146653-d6455e29-598e-46d0-9898-75f5ee5230c9.png)